pilot/pkg/networking/core/v1alpha3/envoyfilter/deprecated.go

```diff
@@ -51,7 +52,9 @@ func DeprecatedInsertUserFilters(in *plugin.InputParams, listener *xdsapi.Listen
 
 	listenerIPAddress := getListenerIPAddress(listener.Address)
 	if listenerIPAddress == nil {
-		log.Warnf("Failed to parse IP Address from plugin listener")
+		debug, _ := json.MarshalIndent(listener, "howardjohn", "  ")
+		log.Errorf("howardjohn: %s", debug)
+		log.Warnf("Failed to parse IP Address from plugin listener: %v", listener.Address)
```

Comment:
can we remove this code completely in 1.6?

Reply:
Oops, missed the debug log. Ill clean that up. Yeah I opened an issue for this, was planning to send out a PR later if no one else does